dask_saturn/core.py

```diff
@@ -41,17 +41,48 @@
 
 
 class SaturnCluster(SpecCluster):
+    """
+    Create a ``SaturnCluster``, an extension of ``distributed.SpecCluster``
+    specific to Saturn Cloud.
+
+    :param n_workers: Number of workers to provision for the cluster. By default,
+        the cluster will start with 0 workers.
+    :param cluster_url: URL for the "cluster" service running in kubernetes. This
+        component of a ``Dask`` cluster in kubernetes knows how to create pods
+        for workers and the scheduler.
+    :param worker_size: A string with the size to use for each worker. A list of
+        valid sizes and their details can be obtained with ``dask_saturn.describe_sizes()``.
+        If no size is provided, this will default to the size configured for Jupyter
+        in your Saturn Cloud project.
+    :param worker_size: A string with the size to use for the scheduler. A list of
+        valid sizes and their details can be obtained with ``dask_saturn.describe_sizes()``.
+        If no size is provided, this will default to the size configured for Jupyter
+        in your Saturn Cloud project.
+    :param nprocs: The number of ``dask-worker`` processes run on each host in a distributed
+        cluster.
+    :param nthreads: The number of vCPUs allocated to each ``dask-worker`` process.
```

Comment:
this actually does not map to vCPUs the way we run dask, because we always allocate an entire node to each pod. vCPUs comes from the worker_size, so this is just number of threads per process.

Reply:
ah I see, ok. updated in https://github.com/saturncloud/dask-saturn/pull/24/commits/2026fcf95639cbf8201f5be9b815d0e04af621bf

Reply:
perfection